### PR TITLE
Add Turku station in Finland

### PR DESCRIPTION
**Pull request by AdamRJensen**
Station was moved approximately 200 m some time between 2019 and 2024 with a minor data gap.

### CODE REVIEW

**Comment by AdamRJensen** [comment]
Elevation taken from Google Earth. Height is of building (60 m) whereas surrounding street level is around 24 m.

File: solarstations.csv [hunk 1]
```diff
@@ -368,3 +368,4 @@ Mt. Waliguan,,,China,36.28749,100.8963,3810,2011-,WMO GAW,,,https://gawsis.meteo
 Longfengshan,,,China,44.7299,127.5999,331,2009-,WMO GAW,,,https://gawsis.meteoswiss.ch/GAWSIS/#/search/station/stationReportDetails/0-20008-0-LFS,,1,Thermopile,G;B;D
 Shangdianzi,SDZ,,China,40.65,117.1166,287,2005-,WMO GAW,China Meteorological Administration,,,,1,Thermopile,G;B;D;IR
 Linan,,,China,30.3000,119.7300,138,2005-,WMO GAW,China Meteorological Administration,,,,1,Thermopile,G;B;D;IR
+Turku,,,Finland,60.4484,22.2976,60,2019-,,New Energy Research Center (NERC),Located ontop of tall building.,,Upon request,Tier 1,Thermopile,G;B;D
```
Comment (IoannisSifnaios) [inline solarstations.csv:371]: Turku,,,Finland,60.4484,22.2976,60,2019-,,New Energy Research Center (NERC),Located ontop of tall building.,,Upon request,1,Thermopile,G;B;D

**Review by IoannisSifnaios — CHANGES_REQUESTED** [review]
Just remove the word "Tier". Otherwise ok!

File: solarstations.csv (hunk 1 above)
Comment (AdamRJensen) [inline solarstations.csv:371]:
```suggestion
Turku,,,Finland,60.4484,22.2976,60,2019-,,New Energy Research Center (NERC),Located ontop of tall building.,,Upon request,1,Thermopile,G;B;D
```

**Pull request merged**